context_test.go

```diff
@@ -1018,7 +1018,9 @@ func TestContextRenderFile(t *testing.T) {
 
 	assert.Equal(t, http.StatusOK, w.Code)
 	assert.Contains(t, w.Body.String(), "func New() *Engine {")
-	assert.Equal(t, "text/plain; charset=utf-8", w.Header().Get("Content-Type"))
```

Comment:
@thinkerou Could you provide some refernce?

Reply:
@appleboy https://github.com/gin-gonic/gin/pull/2638#discussion_r616274265